src/views/PeopleSearch/components/SearchFieldList/SearchFieldList.module.scss

```diff
@@ -7,8 +7,8 @@
     color: var(--mui-palette-primary-main);
   }
 
-  &_text {
-    color: var(--mui-palette-primary-cyan);
+  &_gordon_text {
```

Comment:
Did you change the searchFieldList index file to use this new CSS class name?

Reply:
Yup, I changed the call name accordingly